docs/ui/components/Step/Step.tsx

```diff
@@ -0,0 +1,66 @@
+import { css } from '@emotion/react';
+import { spacing, theme, typography } from '@expo/styleguide';
+import { PropsWithChildren } from 'react';
+
+type Props = PropsWithChildren<{
+  label: string;
+  connected?: boolean;
```

Comment:
Could we rename this `withConnector`? I am iffy about `connected` because it makes it seem like it might be a network connection/React Context or something.